src/index.js

```diff
@@ -17,18 +18,14 @@ app.use(require('morgan')('dev'));
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 app.use('/docs', swaggerUi.serve, swaggerUi.setup(swaggerdoc));
-
-app.use('/', (req, res) => {
-  res.status(200).json({
-    message: `Welcome to the Kifaru backend page`
-  });
-});
+app.use('/api/v1/', routes);
 
 // catch 404 and forward to error handler
-app.use((req, res, next) => {
-  const err = new Error('Not Found');
-  err.status = 404;
-  next(err);
+app.all('*', (req, res) => {
+  res.status(404).json({
+    status: 404,
```

Comment:
this is not required. Status has already been set.

Reply:
Okay, I will remove it.